### PR TITLE
config(docs): add release 6.1 branch config for docs repos

### PR DESCRIPTION
**Pull request by ran-huang**
as title

### CODE REVIEW

**Comment by ran-huang** [comment]
@Mini256 PTAL

**Review by TomShawn — APPROVED** [review]
LGTM

**Review by purelind — APPROVED** [review]
LGTM

File: prow/config/config.yaml [hunk 1]
```diff
@@ -765,6 +774,15 @@ branch-protection:
                   - "license/cla"
                   - "tide"
                 strict: false
+            release-6.1:
+              protect: true
+              required_status_checks:
+                contexts:
+                  - "jenkins-docs/verify"
```
Comment (ran-huang) [inline prow/config/config.yaml:781]:
```suggestion
                  - "jenkins-docs-cn/verify"
```

**Pull request merged**